content/en/docs/contributing/pr-checks.md

```diff
@@ -0,0 +1,79 @@
+[fix content issues automatically](/docs/contributing/#fix-content-issues-automatically)
+by running `npm run test-and-fix` on your machine.
+
+Additionally, you can comment `/fix:all` on your Pull Request. This will make
+the OpenTelemetry Bot run those commands on your behalf and update the PR. Make
+sure that you pull those changes locally.
+
+{{% /alert %}}
+
+## Easy CLA
+
+This check fails if you haven't
+[signed the CLA](/docs/contributing/#sign-the-cla).
+
+## Netlify deployment
+
+If the [Netlify](https://www.netlify.com/) build fails, select **Details** for
+more information.
+
+## Style checks
+
+To make sure that contributions follow our
+[style guide](/docs/contributing/style-guide) we have implemented a set of
+checks that verify style guide rules and fail l if they find any issues.
+
+The following list describes current checks and what you can do to fix related
+errors:
+
+- `TEXT linter`: This check verifies that
+  [OpenTelemetry-specific terms and words are used consistently across the site](/docs/contributing/style-guide#opentelemetryio-word-list).
+  If any issues are found, annotations are added to your files in the
+  `files changed` view of your PR. Fix those to turn the check green. As an
+  alternative, you can run `npm run check:text -- --fix` locally to fix most
+  issues. Run `npm run check:text` again and manually fix the remaining issues.
+- `MARKDOWN linter`: This check verifies that
+  [standards and consistency for Markdown files are enforced](/docs/contributing/style-guide#markdown-standards).
+  If any issues are found, run `npm:run format` to fix most issues. For more
+  complex issues, run `npm run check:markdown` and apply the suggested changes.
+- `SPELLING check`: This check verifies that
+  [all words are spelled correctly](/docs/contributing/style-guide#spell-checking).
+- `CSPELL:IGNORE check`: This check will verify that all words in your cSpell
+  ignore list are normalized. If this check fails, run `npm run fix:dict`
+  locally and push the changes in a new commit.
+- `FILENAME check`: This check verifies that all
+  [files are formatted by prettier](/docs/contributing/style-guide#file-format).
+  If this check fails, run `npm fix:format` locally and push the changes in a
+  new commit.
+- `FILE FORMAT`: This check verifies that all
+  [file names are in kebab-case](/docs/contributing/style-guide#file-names). If
+  this check fails, run `npm fix:filenames` locally and push the changes in a
+  new commit.
+- `BUILD and CHECK LINKS` / `REFCACHE updates?`: This check verifies that all
+  links that your commits are introducing are functional. Run
+  `npm run check:links` to check them locally. This also updates the reference
+  cache, or `REFCACHE`. Push any changes to the `REFCACHE` in a new commit.
+- `WARNINGS in build log?`: If this check fails, review the build log for any
+  other potential issues. Ask maintainers for help, if you are unsure how to
+  recover.
```

Comment:
I would suggest putting each entry into its own H3 section. We could then put the commands in code block, which would make them easier to spot.

That said, this is ok to merge as is IMHO.

Reply:
fixed